src/pkg/services/m365/sites.go

```diff
@@ -145,15 +149,10 @@ func ParseSite(item models.Siteable) *Site {
 				return s
 			}
 
-			s.OwnerID, err = str.AnyValueToString("id", group)
-			if err != nil {
-				return s
-			}
-
-			s.OwnerEmail, err = str.AnyValueToString("email", group)
-			if err != nil {
-				return s
-			}
+			// ignore the errors, these might or might not exist
+			// if they don't exist, we'll just have an empty string
+			s.OwnerID, _ = str.AnyValueToString("id", group)
+			s.OwnerEmail, _ = str.AnyValueToString("email", group)
```

Comment:
the linter should complain here (rightly) about swallowing errors.  The errors should at least get logged.  Or returned, if the error is not a "not found" case.

Reply:
linter def did not complain. Happy to log it -- but is it really an error? we know there's (seemingly valid) cases where either of this might not be returned at all. Why clutter the error output?

Reply:
It could be an error.  And sentinels tend to be easier to manage than pairwise combinatorics of values+flags+errors.  But it doesn't look like I properly built in sentinels here, so an Info level log is probably sufficient.

Reply:
roger. adding this.